README.md

```diff
@@ -6,3 +6,30 @@
 
 ## Overview
 A flexible and easy way to register, manage and group your applications.
+
+## Prerequisities
+
+- [Docker](https://www.docker.com/get-started)
+- [Minikube](https://github.com/kubernetes/minikube) 1.0.1
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) 1.12.5
+- [Helm](https://github.com/kubernetes/helm) 2.10.0
+- [Kyma CLI](https://github.com/kyma-project/cli) master
+
+## Installation
+
+To install Compass locally, run:
+
+```bash
+./installation/scripts/run.sh
```

Comment:
Hmm, in `Installation` I would specify the following cases:

- Deploy on Kyma (just helm install ...)
- Install Compass with Kyma locally (here's your script)

Reply:
Please add subheaders